code/modules/mob/living/carbon/human/death.dm

```diff
@@ -6,6 +6,8 @@
 			new /obj/effect/temp_visual/gib_animation(loc, "gibbed-r")
 		if("plasma")
 			new /obj/effect/temp_visual/gib_animation(loc, "gibbed-h") //This will have more use in the near future
+		if("polysmorph")
+			new /obj/effect/temp_visual/gib_animation(loc, "gibbed-h")
```

Comment:
```suggestion
			new /obj/effect/temp_visual/gib_animation(loc, "gibbed-a")
```
just making sure you don't mean this

Reply:
I was very much on the fence about this. I thought it might be cheap to have the xeno one as a placeholder, but then again it can't be worse than a human placeholder, so suuure